### PR TITLE
Ruslan / Transfer from Daedalus Paper Wallets

### PR DESCRIPTION
**Pull request by vantuz-subhuman**
1. Implemented a functionality to also transfer all coins from a Daedalus Paper Wallet
2. Tried to implement some UI (by example). Now it just adds one additional button to the main "Transfer from Daedalus" screen, so next to "Transfer all funds from Daedalus wallet" it adds "Transfer all funds from Daedalus Paper wallet". If someone has better ideas - they are welcomed very much to contribute!
3. I couldn't figure out on how to make one mnemonics field to accept either 12 or 27 words, so there's a need to have two separate "screens" for inputs either from a regular wallet or from a paper one

### Problems

1. I cannot figure out the translation keys. I have added some new text fields into the code, but if I run `npm run manage-translations` - it only removes a bunch of useful lines from translation files. Help welcomed, plz.

2. I couldn't figure out on how to put these two buttons in a row next to each other, rather than like this in vertical column =\ Help welcomed, plz, too.

image

3. I couldn't figure out on how to make it a single screen for mnemonics, but with like a switcher from regular to mnemonics. But: a) not guaranteed whether it would be better for users; and b) not guaranteed that it would worth the effort.

## UI

image

image

image

Apart from having a separate button for paper-wallets, and the input field reconfigured for 17 words, and a couple of changes in text fields - everything else is the same. Transferring follows absolutely the same workflow.

### CODE REVIEW

File: app/containers/transfer/DaedalusTransferInstructionsPage.js [hunk 1]
```diff
@@ -8,21 +8,27 @@ import TransferInstructionsPage from '../../components/transfer/TransferInstruct
...
+    defaultMessage: '!!!Yoroi and Daedalus wallets use different key derivation scheme and they each have a separate format for addresses. For this reason, Daedalus wallets cannot be restored and continued to be used in Icarus and vice versa. This will change in the future. For now, to use funds from your Daedalus wallet, you need to transfer them to your Icarus wallet. Daedalus and Icarus wallets are fully compatible for transferring of funds. If you don’t have a working copy of Daedalus, you can use your 12-word recovery phrase (or 27-words for a paper wallet) used to restore and transfer the balance from Daedalus into Icarus.',
```
[the remark this answers is not in the document]
Comment (vantuz-subhuman) [reply app/containers/transfer/DaedalusTransferInstructionsPage.js]: Ah yes, I missed couple of more mentions in the same string. Replace what I could find now

**Comment by vantuz-subhuman** [comment]
Here's a video of how it works:

<a href="http://www.youtube.com/watch?feature=player_embedded&v=63TXvEUWC1E
" target="_blank">IMAGE ALT TEXT HERE</a>

**Comment by vantuz-subhuman** [comment]
@nicarq, if this UI is ok for us - then PR is ready for review and merge. Everything works and there are unit-tests for the new feature.

image

**Comment by SebastienGllmt** [comment]
As I mentioned on Slack, this PR is missing the change to the DE and FR translation files. This can be done in a separate PR or in this PR. Either is fine with me.

**Pull request merged**